app/src/main/java/org/p2p/wallet/history/ui/history/HistoryPresenter.kt

```diff
@@ -102,7 +102,7 @@ class HistoryPresenter(
             val deferredBlockChainTransactions = async { fetchBlockChainTransactions(isRefresh) }
             moonpayTransactionsList = deferredSellTransactions.await()
             blockChainTransactionsList += deferredBlockChainTransactions.await()
-            if (moonpayTransactionsList.isFailed && blockChainTransactionsList.isFailed) {
+            if (moonpayTransactionsList.isFailed || blockChainTransactionsList.isFailed) {
```

Comment:
im not sure that's right
if we have moonpayTransactionList but failed blockhain list - we need to show moonpay transactions

Reply:
for now if you disable internet and go to history it will be empty screen not error
it is 146% issue

Reply:
@gslevinkov if there is internet connection issue moonpayTransactionsList.isFailed == false
maybe here is an issue
can you fix it or you know where to fix?

Reply:
check the fetchMoonpayTransaction i guess, but this makes no sense
there will be absolute new logic for presenter, why fixing it now?

Reply:
for QA to check error state is ok in https://p2pvalidator.atlassian.net/browse/PWN-6886
lol

Reply:
error state would be again not ok if you rewrite logic in presenter, Davran or you will just remove all the code here and write own logic
well, if you want to fix it now - okay, i won't stop you, i just asked

Reply:
ok ok lets then just make a build for QA without merge